packages/web-app-files/src/components/SideBar/Shares/FileShares.vue

```diff
@@ -114,9 +114,10 @@ export default {
       transitionGroupActive: false,
 
       // panel types
-      PANEL_SHOW,
-      PANEL_EDIT,
-      PANEL_NEW
+      VIEW_SHOW,
+      VIEW_EDIT,
+      VIEW_NEW,
+      CURRENT_VIEW: VIEW_SHOW
```

Comment:
`currentView` is not a constant. Would like to keep the naming in camelCase then.